### ISSUE TITLE
Nightly Eclipse Che E2E devfile tests are failing on creation of workspace

### ISSUE THREAD

**Issue by dmytro-ndp** [eclipse-che/che#18881]
### Describe the bug
Nightly Eclipse Che E2E devfile tests are failing on creation of workspace on minikube: 
https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/basic-MultiUser-Che-check-e2e-tests-against-k8s/3231/console

```
14:12:03 + docker run --shm-size=1g --net=host --ipc=host -p 5920:5920 -e TS_SELENIUM_BASE_URL=https://che-eclipse-che.10.0.101.42.nip.io -e TS_SELENIUM_LOG_LEVEL=DEBUG -e TS_SELENIUM_MULTIUSER=true -e TS_SELENIUM_USERNAME=admin -e TS_SELENIUM_PASSWORD=admin -e TEST_SUITE=test-all-devfiles -e NODE_TLS_REJECT_UNAUTHORIZED=0 -v /mnt/hudson_workspace/workspace/basic-MultiUser-Che-check-e2e-tests-against-k8s/tests/e2e:/tmp/e2e:Z quay.io/eclipse/che-e2e:nightly
14:12:03 WARNING: Published ports are discarded when using host network mode
14:12:04 + '[' -z https://che-eclipse-che.10.0.101.42.nip.io ']'
14:12:04 + '[' -z test-all-devfiles ']'
14:12:04 + export DISPLAY=:20
14:12:04 + DISPLAY=:20
14:12:04 
14:12:04 #######################
14:12:04 
14:12:04 For remote debug connect to the VNC server 0.0.0.0:5920
14:12:04 
14:12:04 #######################
14:12:04 
14:12:04 + Xvfb :20 -screen 0 1920x1080x24
14:12:04 + echo ''
14:12:04 + echo '#######################'
14:12:04 + echo ''
14:12:04 + echo 'For remote debug connect to the VNC server 0.0.0.0:5920'
14:12:04 + echo ''
14:12:04 + echo '#######################'
14:12:04 + echo ''
14:12:04 + x11vnc -display :20 -N -forever
14:12:04 + export TS_SELENIUM_REMOTE_DRIVER_URL=http://localhost:4444/wd/hub
14:12:04 + TS_SELENIUM_REMOTE_DRIVER_URL=http://localhost:4444/wd/hub
14:12:04 + expectedStatus=200
14:12:04 + currentTry=1
14:12:04 + maximumAttempts=5
14:12:04 + /usr/bin/supervisord --configuration /etc/supervisord.conf
14:12:04 ++ curl -s -o /dev/null -w '%{http_code}' --fail http://localhost:4444/wd/hub/status
14:12:04 Wait selenium server availability ...
14:12:04 + '[' 000 '!=' 200 ']'
14:12:04 + ((  currentTry > maximumAttempts  ))
14:12:04 + echo 'Wait selenium server availability ...'
14:12:04 + curentTry=1
14:12:04 + sleep 1
14:12:05 ++ curl -s -o /dev/null -w '%{http_code}' --fail http://localhost:4444/wd/hub/status
14:12:05 Wait selenium server availability ...
14:12:05 + '[' 000 '!=' 200 ']'
14:12:05 + ((  currentTry > maximumAttempts  ))
14:12:05 + echo 'Wait selenium server availability ...'
14:12:05 + curentTry=2
14:12:05 + sleep 1
14:12:05 ++ curl -s -o /dev/null -w '%{http_code}' --fail http://localhost:4444/wd/hub/status
14:12:05 + '[' 000 '!=' 200 ']'
14:12:05 + ((  currentTry > maximumAttempts  ))
14:12:05 + echo 'Wait selenium server availability ...'
14:12:05 + curentTry=3
14:12:05 + sleep 1
14:12:05 Wait selenium server availability ...
14:12:07 ++ curl -s -o /dev/null -w '%{http_code}' --fail http://localhost:4444/wd/hub/status
14:12:07 + '[' 200 '!=' 200 ']'
14:12:07 + mount
14:12:07 + grep e2e
14:12:07 /dev/vda1 on /tmp/e2e type xfs (rw,relatime,seclabel,attr2,inode64,noquota)
14:12:07 + echo 'The local code is mounted. Executing local code.'
14:12:07 + cd /tmp/e2e
14:12:07 The local code is mounted. Executing local code.
14:12:07 + npm install
14:12:12 
14:12:12 > chromedriver@80.0.1 install /tmp/e2e/node_modules/chromedriver
14:12:12 > node install.js
14:12:12 
14:12:12 ChromeDriver binary exists. Validating...
14:12:12 ChromeDriver is already available at '/tmp/80.0.3987.16/chromedriver/chromedriver'.
14:12:12 Copying to target path /tmp/e2e/node_modules/chromedriver/lib/chromedriver
14:12:12 Fixing file permissions
14:12:12 Done. ChromeDriver binary available at /tmp/e2e/node_modules/chromedriver/lib/chromedriver/chromedriver
14:12:13 npm WARN e2e@1.0.0 No repository field.
14:12:13 npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.2 (node_modules/fsevents):
14:12:13 npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
14:12:13 
14:12:13 added 223 packages from 282 contributors and audited 224 packages in 4.897s
14:12:13 
14:12:13 18 packages are looking for funding
14:12:13   run `npm fund` for details
14:12:13 
14:12:13 found 1 high severity vulnerability
14:12:13   run `npm audit fix` to fix them, or `npm audit` for details
14:12:13 + '[' test-all-devfiles == load-test ']'
14:12:13 + SCREEN_RECORDING=true
14:12:13 + '[' true == true ']'
14:12:13 + echo 'Starting ffmpeg recording...'
14:12:13 + mkdir -p /tmp/ffmpeg_report
14:12:13 Starting ffmpeg recording...
14:12:13 + ffmpeg_pid=196
14:12:13 + trap kill_ffmpeg 2 15
14:12:13 + echo 'Running TEST_SUITE: test-all-devfiles with user: admin'
14:12:13 + npm run test-all-devfiles
14:12:13 + nohup ffmpeg -y -video_size 1920x1080 -framerate 24 -f x11grab -i :20.0 /tmp/ffmpeg_report/output.mp4
14:12:13 Running TEST_SUITE: test-all-devfiles with user: admin
14:12:14 
14:12:14 > e2e@1.0.0 test-all-devfiles /tmp/e2e
14:12:14 > ./generateIndex.sh && npm run lint && npm run tsc && mocha --opts mocha-all-devfiles.opts
14:12:14 
14:12:14 Generating index.ts file...
14:12:15 
14:12:15 > e2e@1.0.0 lint /tmp/e2e
14:12:15 > tslint --fix -p .
14:12:15 
14:12:17 
14:12:17 Could not find implementations for the following rules specified in the configuration:
14:12:17     label-undefined
14:12:17     no-constructor-vars
14:12:17     no-duplicate-key
14:12:17     no-trailing-comma
14:12:17     no-unreachable
14:12:17 Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
14:12:17 If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
14:12:17         
14:12:18 The 'no-string-literal' rule threw an error in '/tmp/e2e/utils/requestHandlers/CheApiRequestHandler.ts':
14:12:18 TypeError: ts.unescapeIdentifier is not a function
14:12:18     at cb (/tmp/e2e/node_modules/tslint/lib/rules/noStringLiteralRule.js:54:39)
14:12:18     at visitNode (/tmp/e2e/node_modules/typescript/lib/typescript.js:16505:24)
14:12:18     at Object.forEachChild (/tmp/e2e/node_modules/typescript/lib/typescript.js:16720:24)
14:12:18     at cb (/tmp/e2e/node_modules/tslint/lib/rules/noStringLiteralRule.js:60:19)
14:12:18     at visitNode (/tmp/e2e/node_modules/typescript/lib/typescript.js:16505:24)
14:12:18     at Object.forEachChild (/tmp/e2e/node_modules/typescript/lib/typescript.js:16751:24)
14:12:18     at cb (/tmp/e2e/node_modules/tslint/lib/rules/noStringLiteralRule.js:60:19)
14:12:18     at visitNodes (/tmp/e2e/node_modules/typescript/lib/typescript.js:16514:30)
14:12:18     at Object.forEachChild (/tmp/e2e/node_modules/typescript/lib/typescript.js:16740:24)
14:12:18     at cb (/tmp/e2e/node_modules/tslint/lib/rules/noStringLiteralRule.js:60:19)
14:12:21 
14:12:21 > e2e@1.0.0 tsc /tmp/e2e
14:12:21 > tsc -p .
14:12:21 
14:12:27 (node:283) DeprecationWarning: Configuration via mocha.opts is DEPRECATED and will be removed from a future version of Mocha. Use RC files or package.json instead.
14:12:27 
14:12:27 ################## Launch Information ##################
14:12:27 
14:12:27       TS_SELENIUM_BASE_URL: https://che-eclipse-che.10.0.101.42.nip.io
14:12:27       TS_SELENIUM_HEADLESS: false
14:12:27 
14:12:27       TS_SELENIUM_USERNAME: admin
14:12:27       TS_SELENIUM_PASSWORD: admin
14:12:27 
14:12:27       TS_SELENIUM_HAPPY_PATH_WORKSPACE_NAME: petclinic-dev-environment
14:12:27       TS_SELENIUM_DELAY_BETWEEN_SCREENSHOTS: 1000
14:12:27       TS_SELENIUM_REPORT_FOLDER: ./report
14:12:27       TS_SELENIUM_EXECUTION_SCREENCAST: false
14:12:27       DELETE_SCREENCAST_IF_TEST_PASS: true
14:12:27       TS_SELENIUM_REMOTE_DRIVER_URL: http://localhost:4444/wd/hub
14:12:27       DELETE_WORKSPACE_ON_FAILED_TEST: false
14:12:27       TS_SELENIUM_LOG_LEVEL: DEBUG
14:12:27 
14:12:27       to output timeout variables, set TS_SELENIUM_PRINT_TIMEOUT_VARIABLES to true
14:12:27  ######################################################## 
14:12:27 
14:12:27           ▼ PreferencesHandler.setConfirmExit to never
14:12:27 
14:12:27   Login test
14:12:27           ▼ DriverHelper.navigateToUrl https://che-eclipse-che.10.0.101.42.nip.io
14:12:28   [WARN] PreferencesHandler.setPreference could not set theia-user-preferences from api/preferences response, forcing manually.
14:12:29           ▼ PreferencesHandler.setTerminalToDom
14:12:29           ▼ MultiUserLoginPage.login
14:12:29           ▼ CheLoginPage.waitEclipseCheLoginFormPage
14:12:30           ▼ CheLoginPage.inputUserNameEclipseCheLoginPage username: "admin"
14:12:31           ▼ CheLoginPage.inputPaswordEclipseCheLoginPage password: "admin"
14:12:31           ▼ CheLoginPage.clickEclipseCheLoginButton
14:12:32     ✓ Login (4529ms)
14:12:32 
14:12:32   C/C++ test
14:12:32     Create C/C++ workspace
14:12:32           ▼ Dashboard.waitPage
14:12:54       1) Open 'New Workspace' page
14:12:54   [ERROR] CheReporter runner.on.fail: C/C++ test Create C/C++ workspace Open 'New Workspace' page failed after 21286ms
14:12:54 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Create_C/C++_workspace_Open_'New_Workspace'_page'
14:12:54     at Object.mkdirSync (fs.js:757:3)
14:12:54     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:12:54     at Runner.emit (events.js:203:15)
14:12:54     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:12:54     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:12:54     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:12:54     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:12:54     at process._tickCallback (internal/process/next_tick.js:68:7)
14:12:54 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
14:12:54 (node:283) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
14:12:54           ▼ Ide.waitAndSwitchToIdeFrame
14:19:00           ▼ Ide.waitPreloaderVisible
14:25:07       2) Wait for workspace readiness
14:25:07   [ERROR] CheReporter runner.on.fail: C/C++ test Create C/C++ workspace Wait for workspace readiness failed after 723212ms
14:25:07 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Create_C/C++_workspace_Wait_for_workspace_readiness'
14:25:07     at Object.mkdirSync (fs.js:757:3)
14:25:07     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:25:07     at Runner.emit (events.js:203:15)
14:25:07     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:25:07     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:25:07     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:25:07     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:25:07     at process._tickCallback (internal/process/next_tick.js:68:7)
14:25:07 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
14:25:07     Test opening file
14:25:07           ▼ ProjectTree.expandPathAndOpenFile "cpp-hello-world" filename: hello.cpp
14:25:07           ▼ ProjectTree.expandPath "cpp-hello-world"
14:25:07           ▼ ProjectTree.expandItem "cpp-hello-world"
14:25:07       3) Expand project and open file in editor
14:25:07   [ERROR] CheReporter runner.on.fail: C/C++ test Test opening file Expand project and open file in editor failed after 3039ms
14:25:07 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Test_opening_file_Expand_project_and_open_file_in_editor'
14:25:07     at Object.mkdirSync (fs.js:757:3)
14:25:07     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:25:07     at Runner.emit (events.js:203:15)
14:25:07     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:25:07     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:25:07     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:25:07     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:25:07     at process._tickCallback (internal/process/next_tick.js:68:7)
14:25:07 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
14:25:07           ▼ Editor.moveCursorToLineAndChar title: "hello.cpp" line: "6" char: "1"
14:25:07           ▼ Editor.performKeyCombination title: "hello.cpp" text: ""
14:25:07       4) Prepare file for LS tests
14:25:07   [ERROR] CheReporter runner.on.fail: C/C++ test Test opening file Prepare file for LS tests failed after 3030ms
14:25:07 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Test_opening_file_Prepare_file_for_LS_tests'
14:25:07     at Object.mkdirSync (fs.js:757:3)
14:25:07     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:25:07     at Runner.emit (events.js:203:15)
14:25:07     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:25:07     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:25:07     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:25:07     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:25:07     at process._tickCallback (internal/process/next_tick.js:68:7)
14:25:07 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
14:25:07     Validation of project build
14:25:07           ▼ TopMenu.selectOption "Terminal"
14:25:07           ▼ TopMenu.clickOnTopMenuButton "Terminal"
14:25:07           ▼ Ide.closeAllNotifications
14:25:07           ▼ NotificationCenter.open
14:25:07           ▼ NotificationCenter.clickIconOnStatusBar
14:25:07       5) Run command 'build'
14:25:07   [ERROR] CheReporter runner.on.fail: C/C++ test Validation of project build Run command 'build' failed after 3039ms
14:25:07 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Validation_of_project_build_Run_command_'build''
14:25:07     at Object.mkdirSync (fs.js:757:3)
14:25:07     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:25:07     at Runner.emit (events.js:203:15)
14:25:07     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:25:07     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:25:07     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:25:07     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:25:07     at process._tickCallback (internal/process/next_tick.js:68:7)
14:25:07 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 5)
14:25:07           ▼ TopMenu.selectOption "Terminal"
14:25:07           ▼ TopMenu.clickOnTopMenuButton "Terminal"
14:25:07           ▼ Ide.closeAllNotifications
14:25:07           ▼ NotificationCenter.open
14:25:07           ▼ NotificationCenter.clickIconOnStatusBar
14:25:09       6) Run command 'run'
14:25:09   [ERROR] CheReporter runner.on.fail: C/C++ test Validation of project build Run command 'run' failed after 3038ms
14:25:09 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Validation_of_project_build_Run_command_'run''
14:25:09     at Object.mkdirSync (fs.js:757:3)
14:25:09     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:25:09     at Runner.emit (events.js:203:15)
14:25:09     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:25:09     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:25:09     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:25:09     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:25:09     at process._tickCallback (internal/process/next_tick.js:68:7)
14:25:09 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 6)
14:25:09     Language server validation
14:25:09           ▼ Editor.type title: "hello.cpp" text: "error_text;"
14:25:09           ▼ Editor.selectTab "hello.cpp"
14:25:09           ▼ Editor.waitTab "hello.cpp"
14:25:14       7) Error highlighting
14:25:14   [ERROR] CheReporter runner.on.fail: C/C++ test Language server validation Error highlighting failed after 5060ms
14:25:14 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Language_server_validation_Error_highlighting'
14:25:14     at Object.mkdirSync (fs.js:757:3)
14:25:14     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:25:14     at Runner.emit (events.js:203:15)
14:25:14     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:25:14     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:25:14     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:25:14     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:25:14     at process._tickCallback (internal/process/next_tick.js:68:7)
14:25:14           ▼ Ide.closeAllNotifications
14:25:14           ▼ NotificationCenter.open
14:25:14           ▼ NotificationCenter.clickIconOnStatusBar
14:25:14 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 7)
14:25:17       8) Suggestion invoking
14:25:17   [ERROR] CheReporter runner.on.fail: C/C++ test Language server validation Suggestion invoking failed after 3052ms
14:25:17 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Language_server_validation_Suggestion_invoking'
14:25:17     at Object.mkdirSync (fs.js:757:3)
14:25:17     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:25:17     at Runner.emit (events.js:203:15)
14:25:17     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:25:17     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:25:17     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:25:17     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:25:17     at process._tickCallback (internal/process/next_tick.js:68:7)
14:25:17 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 8)
14:25:17           ▼ Editor.moveCursorToLineAndChar title: "hello.cpp" line: "15" char: "9"
14:25:17           ▼ Editor.performKeyCombination title: "hello.cpp" text: ""
14:25:20       9) Autocomplete
14:25:20   [ERROR] CheReporter runner.on.fail: C/C++ test Language server validation Autocomplete failed after 3023ms
14:25:20 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Language_server_validation_Autocomplete'
14:25:20     at Object.mkdirSync (fs.js:757:3)
14:25:20     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:25:20     at Runner.emit (events.js:203:15)
14:25:20     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:25:20     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:25:20     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:25:20     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:25:20     at process._tickCallback (internal/process/next_tick.js:68:7)
14:25:20 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 9)
14:25:20     Stopping and deleting the workspace
14:25:20           ▼ Dashboard.stopWorkspaceByUI "get-started"
14:25:20           ▼ Dashboard.openDashboard
14:25:20           ▼ Dashboard.waitPage
14:25:42       10) Stop worksapce
14:25:42   [ERROR] CheReporter runner.on.fail: C/C++ test Stopping and deleting the workspace Stop worksapce failed after 20540ms
14:25:42 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Stopping_and_deleting_the_workspace_Stop_worksapce'
14:25:42     at Object.mkdirSync (fs.js:757:3)
14:25:42     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:25:42     at Runner.emit (events.js:203:15)
14:25:42     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:25:42     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:25:42     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:25:42     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:25:42     at process._tickCallback (internal/process/next_tick.js:68:7)
14:25:42 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 10)
14:25:42           ▼ Dashboard.deleteWorkspaceByUI "get-started"
14:25:42           ▼ Dashboard.openDashboard
14:25:42           ▼ Dashboard.waitPage
14:26:04       11) Remove workspace
14:26:04   [ERROR] CheReporter runner.on.fail: C/C++ test Stopping and deleting the workspace Remove workspace failed after 20483ms
14:26:04 (node:283) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir './report/C/C++_test_Stopping_and_deleting_the_workspace_Remove_workspace'
14:26:04     at Object.mkdirSync (fs.js:757:3)
14:26:04     at Runner.<anonymous> (/tmp/e2e/driver/CheReporter.ts:148:12)
14:26:04     at Runner.emit (events.js:203:15)
14:26:04     at Runner.fail (/tmp/e2e/node_modules/mocha/lib/runner.js:310:8)
14:26:04     at /tmp/e2e/node_modules/mocha/lib/runner.js:698:18
14:26:04     at done (/tmp/e2e/node_modules/mocha/lib/runnable.js:335:5)
14:26:04     at /tmp/e2e/node_modules/mocha/lib/runnable.js:406:11
14:26:04     at process._tickCallback (internal/process/next_tick.js:68:7)
14:26:04 (node:283) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 11)
14:26:04 
14:26:04   Test .NET Core
14:26:04     Create .NET Core workspace
14:26:04           ▼ Dashboard.waitPage
14:26:22       12) Open 'New Workspace' page
14:26:22   [ERROR] CheReporter runner.on.fail: Test .NET Core Create .NET Core workspace Open 'New Workspace' page failed after 20178ms
14:26:22           ▼ Ide.waitAndSwitchToIdeFrame
14:32:29           ▼ Ide.waitPreloaderVisible
14:38:35       13) Wait for workspace readiness
14:38:35   [ERROR] CheReporter runner.on.fail: Test .NET Core Create .NET Core workspace Wait for workspace readiness failed after 722970ms
14:38:35     Test opening file
14:38:35           ▼ ProjectTree.expandPathAndOpenFile "dotnet-web-simple" filename: Program.cs
14:38:35           ▼ ProjectTree.expandPath "dotnet-web-simple"
14:38:35           ▼ ProjectTree.expandItem "dotnet-web-simple"
14:38:35       14) Expand project and open file in editor
14:38:35   [ERROR] CheReporter runner.on.fail: Test .NET Core Test opening file Expand project and open file in editor failed after 3024ms
14:38:35           ▼ Editor.moveCursorToLineAndChar title: "Program.cs" line: "18" char: "6"
14:38:35           ▼ Editor.performKeyCombination title: "Program.cs" text: ""
14:38:35       15) Prepare file for LS tests
14:38:35   [ERROR] CheReporter runner.on.fail: Test .NET Core Test opening file Prepare file for LS tests failed after 3021ms
14:38:35     Installing dependencies
14:38:35           ▼ TopMenu.selectOption "Terminal"
14:38:35           ▼ TopMenu.clickOnTopMenuButton "Terminal"
14:38:35           ▼ Ide.closeAllNotifications
14:38:35           ▼ NotificationCenter.open
14:38:35           ▼ NotificationCenter.clickIconOnStatusBar
14:38:35       16) Run command 'update dependencies'
14:38:35   [ERROR] CheReporter runner.on.fail: Test .NET Core Installing dependencies Run command 'update dependencies' failed after 3036ms
14:38:35           ▼ Ide.closeAllNotifications
14:38:35           ▼ NotificationCenter.open
14:38:35           ▼ NotificationCenter.clickIconOnStatusBar
14:38:36       17) Close the terminal tasks
14:38:36   [ERROR] CheReporter runner.on.fail: Test .NET Core Installing dependencies Close the terminal tasks failed after 3036ms
14:38:36     Validation of workspace build
14:38:36           ▼ TopMenu.selectOption "Terminal"
14:38:36           ▼ TopMenu.clickOnTopMenuButton "Terminal"
14:38:36           ▼ Ide.closeAllNotifications
14:38:36           ▼ NotificationCenter.open
14:38:36           ▼ NotificationCenter.clickIconOnStatusBar
14:38:39       18) Run command 'build'
14:38:39   [ERROR] CheReporter runner.on.fail: Test .NET Core Validation of workspace build Run command 'build' failed after 3030ms
14:38:39           ▼ Ide.closeAllNotifications
14:38:39           ▼ NotificationCenter.open
14:38:39           ▼ NotificationCenter.clickIconOnStatusBar
14:38:43       19) Close the terminal tasks
14:38:43   [ERROR] CheReporter runner.on.fail: Test .NET Core Validation of workspace build Close the terminal tasks failed after 3039ms
14:38:43     Run .NET Core example application
14:38:43           ▼ TopMenu.selectOption "Terminal"
14:38:43           ▼ TopMenu.clickOnTopMenuButton "Terminal"
14:38:43           ▼ Ide.closeAllNotifications
14:38:43           ▼ NotificationCenter.open
14:38:43           ▼ NotificationCenter.clickIconOnStatusBar
14:38:45       20) Run command 'run' expecting notification pops up
14:38:45   [ERROR] CheReporter runner.on.fail: Test .NET Core Run .NET Core example application Run command 'run' expecting notification pops up failed after 3044ms
14:38:45     Language server validation
14:38:45           ▼ Ide.closeAllNotifications
14:38:45           ▼ NotificationCenter.open
14:38:45           ▼ NotificationCenter.clickIconOnStatusBar
14:38:49       21) Suggestion invoking
14:38:49   [ERROR] CheReporter runner.on.fail: Test .NET Core Language server validation Suggestion invoking failed after 3018ms
14:38:49           ▼ Editor.type title: "Program.cs" text: "error_text;"
14:38:49           ▼ Editor.selectTab "Program.cs"
14:38:49           ▼ Editor.waitTab "Program.cs"
14:38:54       22) Error highlighting
14:38:54   [ERROR] CheReporter runner.on.fail: Test .NET Core Language server validation Error highlighting failed after 5047ms
14:38:54           ▼ Editor.moveCursorToLineAndChar title: "Program.cs" line: "22" char: "27"
14:38:54           ▼ Editor.performKeyCombination title: "Program.cs" text: ""
14:38:56       23) Autocomplete
14:38:56   [ERROR] CheReporter runner.on.fail: Test .NET Core Language server validation Autocomplete failed after 3015ms
14:38:56     Stopping and deleting the workspace
14:38:57           ▼ Dashboard.stopWorkspaceByUI "get-started"
14:38:57           ▼ Dashboard.openDashboard
14:38:57           ▼ Dashboard.waitPage
14:39:19       24) Stop worksapce
14:39:19   [ERROR] CheReporter runner.on.fail: Test .NET Core Stopping and deleting the workspace Stop worksapce failed after 20561ms
14:39:19           ▼ Dashboard.deleteWorkspaceByUI "get-started"
14:39:19           ▼ Dashboard.openDashboard
14:39:19           ▼ Dashboard.waitPage
14:39:41       25) Remove workspace
14:39:41   [ERROR] CheReporter runner.on.fail: Test .NET Core Stopping and deleting the workspace Remove workspace failed after 20666ms
14:39:41 
14:39:41   Go test
14:39:41     Create Go workspace
14:39:41   [WARN] Manually setting a preference for golang devfile LS based on issue: https://github.com/eclipse/che/issues/16113
14:39:41           ▼ PreferencesHandler.setUseGoLanguageServer to true.
14:39:41       ✓ Workaround for issue #16113 (293ms)
14:39:41           ▼ Dashboard.waitPage
14:39:59       26) Open 'New Workspace' page
14:39:59   [ERROR] CheReporter runner.on.fail: Go test Create Go workspace Open 'New Workspace' page failed after 20156ms
14:39:59           ▼ Ide.waitAndSwitchToIdeFrame
14:46:06           ▼ Ide.waitPreloaderVisible
...
```

![screenshot-Open_'New_Workspace'_page](https://user-images.githubusercontent.com/1197777/105713656-11fa2680-5f24-11eb-985c-a36a360dd775.png)

Fixup: https://github.com/eclipse/che/pull/18428

### Che version

 - [ ] latest
 - [x] nightly
 - [ ] other: please specify

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [x] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [x] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] other: please specify

**Comment by Katka92** [comment]
This issue is probably related to the new Dashboard and should be fixed with https://github.com/eclipse/che/pull/18428 merged.

**Comment by nickboldt** [comment]
Slip to 7.26 since not completed for 7.25

**Comment by dmytro-ndp** [comment]
Fixed in Eclipse Che 7.26.
